### ISSUE TITLE
removeConstraint() causes crash

### ISSUE THREAD

**Issue by IzaZed** [UPBGE/upbge#1116]
Maybe I'm doing something wrong, but whenever I try to remove a constraint, the game engine crashes.

Test File - Press "F" to run the remove Constraint script:
[HingeTEst.zip](https://github.com/UPBGE/upbge/files/4165180/HingeTEst.zip)

Tried the same setup with UPBGE 0.2.4, crashes too

**Comment by youle31** [comment]
I confirm the issue.

I pushed a branch to try to fix it. IMO this comes from the way the constraint are created with NewProxy(true) (just a feeling)
Anyway, I'll push a branch inspired by old bge code which had not this issue and which seem to fix the issue.

Else, in your demo file, this is weird, you mix both Rigid Body joints and bge.constraint and you call create contraint each frame. A bge.constraint only needs to be created 1 time (not each frame) and doesn't need Rigid body joint.

I modified your test file here:

http://pasteall.org/blend/index.php?id=52879

Pull request here: https://github.com/UPBGE/upbge/pull/1118
